app/components/updated_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# A component for displaying when a record was updated.
+class UpdatedComponent < ViewComponent::Base
+  attr_reader :description, :updated_at
+
```

Comment:
**suggestion (non-blocking):** +I like this a lot, but can you add a preview for it so that it is easy to find if we are looking for it?

Reply:
Added in 18fb25e3483596a49c608dc46c33935c3c888a3d.